test/test_transformer.py

```diff
@@ -66,6 +68,49 @@ def get_layers(module):
     return layers
 
 
+def pipe_to_sd(pipe):
+    sd = {}
+    for stage_idx in range(pipe.num_stages):
+        stage_mod = pipe.get_stage_module(stage_idx)
+        sd[f"stage_{stage_idx}"] = stage_mod
+    return sd
+
+with tempfile.TemporaryDirectory() as tmpdir:
+    #Simulate saving the pipe
+    # Option 1:
+    # for stage_idx in range(pipe.num_stages):
+    #     print(f"Saving pipeline stage {stage_idx}")
+    #     stage_mod = pipe.get_stage_module(stage_idx)
+    #     dcp.save(
+    #         {f"stage_{stage_idx}": stage_mod},
```

Comment:
Curious, is the dict required by API of DCP? Can a user directly save `stage_mod`?

Reply:
why does this matter? i think the DCP api had reasons for interfacing with dict instead of model, adding a new variant that takes model and gets its dict should be possible, but i think it's clearer this way that the only part of the model that gets saved is the dict

Reply:
Just to be clear: I like saving the state dict too (instead of the module). That's more composable to me. 
My question above is: is `{f"stage_{stage_idx}": stage_mod}` necessary?